docs/overview_andy

```diff
@@ -0,0 +1,52 @@
+---
+layout: page
+---
+
+# To-do service API
+
+This to-do service provides a mock REST API to create and manage different tasks for users. 
+
+Admins will be able to enroll new users, delegate tasks, and more.
+
```

Comment:
Could you add some use-cases that might help readers see the solutions they're looking for, perhaps?